schemas/mw-article-delete.js

```diff
@@ -0,0 +1,49 @@
+{
+    "title": "MediaWiki Article Delete",
+    "description": "Represents a MW Article Delete Edit event",
+    "type": "object",
+    "properties": {
+        // global event fields
+        "uri": {
+            "type": "string",
+            "format": "uri",
+            "description": "the unique URI identifying the event"
+        },
+        "request_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique UUID v1 ID of the event derived from the X-Request-Id header"
+        },
+        "dt": {
+            "type": "string",
+            "format": "date-time",
+            "description": "the time stamp of the event, in ISO8601 format"
+        },
+        "domain": {
+            "type": "string",
+            "description": "the domain the event pertains to"
+        },
+        // event-specific fields
+        "title": {
```

Comment:
Event if it is redundant, we like longer names so the fields make sense and are easy to understand outside of a particular context.  This is relevant here since JSON Schema has a special `title` field that we are using.